Document/0x06i-Testing-Code-Quality-and-Build-Settings.md

```diff
@@ -171,7 +171,8 @@ These libraries can lead to unwanted side-effects:
 - A library can no longer be maintained or hardly be used, which is why no vulnerabilities are reported and/or fixed. This can lead to having bad and/or vulnerable code in your application through the library.
 - A library can use a license, such as LGPL2.1, which requires the application author to provide access to the source code for those who use the application and request insight in its sources. In fact the application should then be allowed to be redistributed with modifications to its source code. This can endanger the intellectual property (IP) of the application.
 
-Note: there are two widely used package management tools: Carthage and CocoaPods.
+Note: there are three widely used package management tools: Swift Package Manager, Carthage, and CocoaPods.
```

Comment:
this note is kind of 'in the middle" here. What if we put it at the start of the section? Or at the end, bringing some more context to to. I think it's here to justify the following sections talking about those package managers.

Reply:
Yeah I agree, I'll move this to the beginning and add a better introduction of these package mangers. Thanks!

Reply:
@cpholguera I made a quick introduction to the three package managers at the beginning of the section, please let me know your thoughts.